packages/block-library/src/comments/save.js

```diff
@@ -5,7 +5,11 @@ import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
 
 export default function CommentsSave( { attributes: { tagName: Tag } } ) {
 	return (
-		<Tag { ...useBlockProps.save() }>
+		<Tag
+			{ ...useBlockProps.save( {
+				className: 'wp-block-comments-query-loop',
```

Comment:
Is this the best way to add a previous className for backward compatibility?

Reply:
This will add it to _all_ Comments blocks, including newly inserted ones.

Overall, I think I'd prefer if we only added the old class name to blocks that are converted from the old ID -- i.e. somewhere in the deprecation (or migration).

However, I've spent half of the afternoon experimenting with that approach and still haven't found a solution that doesn't cause a block validation error 😅

Reply:
This seems to do the trick: f937d9f